packages/react-charting/docs/colors.md

```diff
@@ -66,7 +66,10 @@ The users can pick colors from our data visualization color palette, which is ca
 
 ### Qualitative Color Palette
 
-![Qualitative Color Palette](images/colors/1.png)
+![First 10 colors in Qualitative palette](images/colors/1.png)
+![Second 10 colors in Qualitative palette](images/colors/2.png)
```

Comment:
slot 20 is named as slot 10. I have asked Karina to fix it.

Reply:
okay

Reply:
This is fixed. Can you refresh the image

Reply:
done